docs/install-and-run.md

```diff
@@ -3,115 +3,125 @@
 
-     Note: If you recently created the Schemes, you need to run them once for them to be picked up by the
-     `pod install` command. If your **Test Target(s)** do not contain these changes to the
-     [Scheme](#scheme-changes) and [Build Phases](#build-phase-changes) after running
-     `pod install`, please re-run `pod install` again.
+And click on **Close**.
 
 #### Step 2: Add EarlGrey as a framework dependency
 
-  1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
-  2. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
-     in your `post_install` actions to add Build Settings and Scheme changes to the **Test Target**
-     for embedding EarlGrey in your app. To use the setup gem, use `gem install earlgrey` before
-     `pod install`.
+1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
 
-  Use the patten shown in the following Podfile to setup EarlGrey with the latest CocoaPods:
+```ruby
+target TEST_TARGET do
+  project PROJECT_NAME
 
-  ```ruby
-  PROJECT_NAME = 'SampleApp'
-  TEST_TARGET = 'SampleAppTests'
-  SCHEME_FILE = 'SampleAppTests.xcscheme'
+  use_frameworks! # Required for Swift Test Targets only
+  inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+  pod 'EarlGrey'
+end
+```
+
+2. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
+in your Podfile's `post_install` section as in below.
 
-  target TEST_TARGET do
-    project PROJECT_NAME
+```ruby
+post_install do |installer|
+  require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+  configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-    use_frameworks! # Required for Swift Test Targets only
-    inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
-    pod 'EarlGrey'
-  end
+3. Your complete Podfile when using EarlGrey with the latest CocoaPods should look like:
+
+```ruby
+PROJECT_NAME = 'SampleApp'
+TEST_TARGET = 'SampleAppTests'
+SCHEME_FILE = 'SampleAppTests.xcscheme'
+
+target TEST_TARGET do
+  project PROJECT_NAME
+
+  use_frameworks! # Required for Swift Test Targets only
+  inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+  pod 'EarlGrey'
+end
+
+post_install do |installer|
+  require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+  configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-  post_install do |installer|
-    require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
-    configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
-  end
-  ```
+4. Install the EarlGrey gem by doing `gem install earlgrey` and then do the `pod install`.
```

Comment:
Move gem install out to first step.

Then in the step where user is asked to do configure_for_earlgrey, tell them they do it for each target / Scheme.

The very last step should be pod install along with this note:
For compatibility between different versions, please see this doc. To download a particular version of the gem, use gem install earlgrey -v x.y.z.